tfrog-motordriver/controlVelocity.h

```diff
@@ -123,6 +124,7 @@ typedef struct _DriverParam
     BOARD_R6B
   } board_version;
   unsigned char protocol_version;
+  unsigned int rs485_baudrate;
```

Comment:
This is struct for on-memory configuration.
Configurations stored in EEPROM is defined in `eeprom.h`.

example of extended command to set config stored in EEPROM:
https://github.com/T-frog/tf-2md3-firmware/blob/f9666a8655c0627d5fa409f4530eaea4484ada10/tfrog-motordriver/communication.c#L1290-L1296